src/textual/css/_help_text.py

```diff
@@ -3,10 +3,11 @@
 from dataclasses import dataclass
 from typing import Iterable, Sequence
 
-from textual._typing import Literal
-from textual.color import ColorParseError
-from textual.css._help_renderables import Example, Bullet, HelpText
-from textual.css.constants import (
+from typing_extensions import Literal
```

Comment:
Not an issue with these changes, but while scanning through this file, and while we're in typing mode, I noticed a couple of instances of `thing: type = None` in parameters where it should be `thing: type | None=None`. Might be worth tweaking them while we're here. (if not note to self to do this).

Reply:
Done.